### ISSUE TITLE
Abandoned Project?

### ISSUE THREAD

**Issue by Slumber86** [wildabeast/BarcodeScanner#243]
There are a big number of issues and PR. And the project is not updated by months. This plugin deserves better mantainers

**Comment by SingleID** [comment]
those who have the skills for an "official" fork are asked to come forward.

**Comment by mattgrande** [comment]
I'd recommend basically anyone who's done a PR recently.

**Comment by edewit** [comment]
We at AeroGear could take over this project as our OTP cordova plugin depends on this @wildabeast how about moving this to AeroGear?

**Comment by wildabeast** [comment]
Agreed! It does need better maintainers. This repo has just been forked to [phonegap/phonegap-plugin-barcodescanner](https://github.com/phonegap/phonegap-plugin-barcodescanner). Makes more sense than an individual / proprietary org I think. Should be able to get this brought up to date in the next bit now.
